resources/assets/lib/circular-progress.tsx

```diff
@@ -31,6 +33,10 @@ export class CircularProgress extends React.PureComponent<Props, any> {
       return null;
     }
 
+    const displayNumber = this.props.reverse
+      ? this.props.current
+      : this.props.max - this.props.current;
```

Comment:
See https://github.com/ppy/osu-web/pull/7772#issuecomment-866544855 on fixing the tooltip instead.

Reply:
Wait, without this changes, the number will count up from `0` to `max` if we use `votesLeft`, because of `max - votesLeft`.

Reply:
The number indicates remaining votes, the fill indicates how many votes are placed. It's the same behavior across other CircularProgress uses.

Reply:
Hmm okay.